tests/portability_test_gold.json

```diff
@@ -0,0 +1,22 @@
+{
+    "VkPhysicalDevicePortabilitySubsetFeaturesKHR": {
+        "constantAlphaColorBlendFactors": 1,
+		"events": 1,
+		"imageViewFormatReinterpretation": 0,
+		"imageViewFormatSwizzle": 1,
+		"imageView2DOn3DImage": 1,
+		"multisampleArrayImage": 1,
+		"mutableComparisonSamplers": 1,
+		"pointPolygons": 0,
+		"samplerMipLodBias": 0,
+		"separateStencilMaskRef": 1,
+		"shaderSampleRateInterpolationFunctions": 1,
+		"tessellationIsolines": 1,
+		"tessellationPointMode": 1,
+		"triangleFans": 1,
+		"vertexAttributeAccessBeyondStride": 1
+    },
+    "VkPhysicalDevicePortabilitySubsetPropertiesKHR": {
+        "minVertexInputBindingStrideAlignment": 8
+    }
+}
```

Comment:
Seems to be missing a termianl newline, where almost all of the other files have one.